go.mod

```diff
@@ -12,11 +12,12 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.14.5
 	github.com/kat-co/vala v0.0.0-20170210184112-42e1d8b61f12
+	github.com/levenlabs/golib v0.0.0-20180911183212-0f8974794783
```

Comment:
can we get a go mod tidy this and the mongo driver below are not used in project